README.md

```diff
@@ -62,6 +62,7 @@
 - [liununu](https://github.com/liununu/liununu)
 - [Filippo Rossi (qu4k)](https://github.com/qu4k/qu4k)
 - [Moe Poi ~](https://github.com/moepoi/moepoi)
+- [Stanley Lim (Spiderpig86)](https://github.com/Spiderpig86)
```

Comment:
Change it from `https://github.com/Spiderpig86` to `https://github.com/Spiderpig86/Spiderpig86`

Reply:
Resolved.